otter/controller.py

```diff
@@ -174,6 +204,10 @@ def delete_group(log, trans_id, group, force):
     """
 
     def check_and_delete(_group, state):
+        if state.paused:
```

Comment:
last discussion I remember I *think* we concluded that it should just be fine to delete a paused group; is there some specific reason we did it this way?